homeassistant/components/tuya/cover.py

```diff
@@ -89,21 +86,21 @@ class TuyaCoverEntityDescription(CoverEntityDescription):
     "ckmkzq": (
         TuyaCoverEntityDescription(
             key=DPCode.SWITCH_1,
-            name="Door",
+            translation_key="door",
```

Comment:
This is a garage door opener, I think it make more sense to use / standardize on the values provided by the device class.